### PR TITLE
Removed Uint256AddLow hint

### PR DESCRIPTION
**Pull request by Sh0g0-1758**
Fixes: #450

### CODE REVIEW

File: pkg/hintrunner/zero/zerohint_uint256.go [hunk 1]
```diff
@@ -19,21 +19,13 @@ import (
...
-// Uint256AddLow hint only computes the sum of the `low` parts of
-// two `uint256` variables and checks for overflow
-func newUint256AddHint(a, b, carryLow, carryHigh hinter.ResOperander, lowOnly bool) hinter.Hinter {
+func newUint256AddHint(a, b, carryLow, carryHigh hinter.ResOperander) hinter.Hinter {
 	name := "Uint256Add"
-	if lowOnly {
-		name += "Low"
-	}
 	return &GenericZeroHinter{
 		Name: name,
```
Comment (har777) [inline pkg/hintrunner/zero/zerohint_uint256.go:25]: you can just make it `Name: "Uint256Add"` and remove the extra variable.

**Review by har777 — APPROVED** [review]
The rest lgtm!

**Comment by TAdev0** [comment]
@Sh0g0-1758  agree with Hari, just remove the extra variable and i'll approve after that :)

**Review by TAdev0 — CHANGES_REQUESTED** [review]
@Sh0g0-1758 agree with Hari, just remove the extra variable and i'll approve after that :)

File: pkg/hintrunner/zero/zerohint_uint256.go (hunk 1 above)
Comment (Sh0g0-1758) [reply pkg/hintrunner/zero/zerohint_uint256.go]: done.

**Comment by Sh0g0-1758** [comment]
@TAdev0 , updated accordingly.

**Review by TAdev0 — APPROVED** [review]

**Pull request merged**